Detectors/Base/src/Detector.cxx

```diff
@@ -171,6 +171,11 @@ void Detector::addAlignableVolumes() const
   LOG(warning) << "Alignable volumes are not yet defined for " << GetName();
 }
 
+void Detector::fillParallelWorld() const
+{
+  LOG(warning) << "Parallel world is not filled for " << GetName();
+}
```

Comment:
wouldn't issue a warning. It's not dangerous if someone doesn't add to para world. What about a simple empty implementation?

Reply:
thanks, I updated this